.github/workflows/ci.yml

```diff
@@ -108,3 +109,87 @@ jobs:
           sudo ls -al $VOLUME_PATH
           sudo tree $VOLUME_PATH
           http --follow --timeout 30 --check-status --pretty format --print hb http://localhost:8080/pulp/api/v3/status/ || true
+
+  pulp-operator:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up Python 3.8
```

Comment:
Can we use a later python like the rest of the workflow? The images run 3.8, but the machines running them don't have to.

Reply:
```suggestion
    - name: Set up Python 3.11
```